### PR TITLE
#2377 show students and mentors event status and status disclaimers

### PR DESCRIPTION
**Pull request by stenington**
This PR for #2377 shows mentors and students the event officiality status when they're looking at an event, and it adds some disclaimers to the site as well about when that status gets finalized. 

I'm opening this PR without tests with the understanding that I might need to write some, but I wanted to get some guidance on that. @shaunxp20 do you think it would be appropriate to write some feature specs here that navigate some of the event selection process for mentors and students? I don't see any existing ones to build on, so maybe it would be good to get some started.

### CODE REVIEW

File: app/views/mentor/dashboards/onboarded/_regional_pitch_events.html.erb [hunk 1]
```diff
@@ -5,7 +5,9 @@
   pitch to a live panel of judges!
 </p>
 
-<div class="step-actions">
+<%= render 'regional_pitch_events/dashboard_disclaimer' %>
```
Comment (shaunxp20) [inline app/views/mentor/dashboards/onboarded/_regional_pitch_events.html.erb]: I was always wondering if there is a convention around when we use single vs double quotes. #nbd-jc (custom acronym i just made up for: no big deal, just curious) 😄 😁 😄

Comment (stenington) [reply app/views/mentor/dashboards/onboarded/_regional_pitch_events.html.erb]: I don't have one, do you have a proposal? A quick google search does come back with lots of "always use double quotes" type discussions so maybe that's a habit I need to pick up?

**Comment by stenington** [comment]
Yeah I don't disagree.

File: app/views/mentor/dashboards/onboarded/_regional_pitch_events.html.erb (hunk 1 above)
Comment (shaunxp20) [reply app/views/mentor/dashboards/onboarded/_regional_pitch_events.html.erb]: Well, my personal preference (which is in the minority from my experience) is to always use single quotes, and then use double quotes if you need to do string interpolation (or need to use single quotes inside the string, or some other reason). This provides helpful information, imho, double quotes - there's something different about this string, whereas double quotes everywhere, no extra information. Having said all of that, I'm (like w/ merging branches in) more a fan of consistency. And fwict, it looks like we use double quotes everywhere. But honestly, either way fine w/ me. #nbd! 👍 😄 😁 😃

Comment (stenington) [reply app/views/mentor/dashboards/onboarded/_regional_pitch_events.html.erb]: Ha ha, yeah looks like [Thoughtbot](https://github.com/thoughtbot/guides/tree/master/style/ruby) and [Github](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#strings) ruby style guides both prefer double quotes. 

I guess I don't actually have a preference on which way to do it, my only preference is that something like rubocop enforces it so I don't have to police myself, which I will constantly forget to do for this kind of detail.

Comment (shaunxp20) [reply app/views/mentor/dashboards/onboarded/_regional_pitch_events.html.erb]: +1 for rubocop enforcement. 💂 🙅 🛑

**Pull request merged**